src/components/Blogs/BlogList.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import BlogItem from './BlogItem';
+import { Button } from "../Button/Button"
+import styles from "./BlogList.module.scss";
+import bimg1 from "../../assets/images/Hospital-Emergency-Block.webp";
+import bimg2 from "../../assets/images/blogimage2.webp";
+import bimg3 from "../../assets/images/blogimage3.webp"
+import iconImg from "../../assets/images/blogUserDummyImage.svg"
+
+const BlogList = () => {
+  return (
+    <div className={styles.container}>
+      <h2>Graana Blogs</h2>
+      <p>Trends and perspectives for buyers and investors</p>
+      <div className={styles.blog_list}>
+        <BlogItem
```

Comment:
These should be coming from an array. We can put the blogs in data/index.js. And then use .map() here which will return `<BlogItem />`

Reply:
solved